src/poller.ts

```diff
@@ -78,7 +78,11 @@ export class Poller {
     };
 
     // Watch the vault spl token account
-    connection.onAccountChange(new PublicKey(address), callback);
+    try{
+      connection.onAccountChange(new PublicKey(address), callback);
```

Comment:
dont think we need to poll above since we would notice a crash immediately at startup. Not sure whether doing it here too also applies since I dont know if the control flow hangs here or if it spawns a new thread. Either way, it is harmless

Reply:
Process of elimination, I guess.  If we see more we can try above